projects/endurance-bridge/index.js

```diff
@@ -0,0 +1,15 @@
+const { sumTokensExport, nullAddress } = require('../helper/unwrapLPs');
+
+const config = {
+  ace: {
+    bridges: [
+      '0xf3310e3f0D46FF5EE7daB69C73452D0ff3979Bed',
```

Comment:
this address is not a smart contract?

Reply:
This address definitely is a smart contract; it's just a display error on the explorer (Blockscout) because this contract is in genesis.json (we processed a major network upgrade).

Overall, for the network nodes, it is a smart contract.